### ISSUE TITLE
Editable panels

### ISSUE THREAD

**Issue by eartharoid** [discord-tickets/bot#362]
Add another form on the panels settings page to load values from a message URL, so panels don't need to be re-created every time something needs changing (like when staff applications are opened/closed).

![image](https://user-images.githubusercontent.com/20905071/206519496-999cbce2-683e-4559-836c-e1f7f657fea9.png)

**Comment by silvershardd-bit** [comment]
omg yes, good idea. is this gonna come this year or?

**Comment by egelhaus** [comment]
> omg yes, good idea. is this gonna come this year or?

@silvershardd-bit , nope, atleast it's very unlikely. DT hasn't gotten a real feature in a while now, only sec fixes.
However, many forks (like mine) have implemented this feature.

**Comment by silvershardd-bit** [comment]
> > omg yes, good idea. is this gonna come this year or?
> 
> [@silvershardd-bit](https://github.com/silvershardd-bit) , nope, atleast it's very unlikely. DT hasn't gotten a real feature in a while now, only sec fixes. However, many forks (like mine) have implemented this feature.

@egelhaus damn alright, thank you. ill have a look at your fork <3
